lsp-ui-doc.el

```diff
@@ -1178,11 +1180,12 @@ It is supposed to be called from `lsp-ui--toggle'"
 (defun lsp-ui-doc-glance ()
   "Trigger display hover information popup and hide it on next typing."
   (interactive)
-  (let ((lsp-ui-doc-show-with-cursor t))
-    (lsp-ui-doc--make-request))
+  (lsp-ui-doc-show)
   (when lsp-ui-doc--unfocus-frame-timer
     (cancel-timer lsp-ui-doc--unfocus-frame-timer))
-  (add-hook 'post-command-hook 'lsp-ui-doc--glance-hide-frame))
+  (run-at-time 1 nil    ;; Since we want hiding after *next* command, not *this* command
```

Comment:
The `run-at-time` is scheduling the lambda on the next main loop already. Can we make this `0` instead of `1`, so the hide will be triggered immediately?

Reply:
Done.